app/controllers/application_controller.rb

```diff
@@ -3,13 +3,7 @@
 class ApplicationController < ActionController::API
 
   def es_client
```

Comment:
I think this method is now unused

Reply:
@jpslav yeah..there's lots of cleanup we'll be doing as this code evolves.